### ISSUE TITLE
Time to reach always has a value

### ISSUE THREAD

**Issue by rmburg** [HULKs/hulk#1325]
It seems like fallen robots often calculate a short TTRKP and thus claim striker. Investigate.

Examples:
- 18.07. game vs. NomadZ, Robot 36:5, during walk-in at the start of the first half
- 17.07. game vs. Nao Devils, Robot 34:4, at 9:25 remaining in the first half

**Comment by MaikRe** [comment]
Because of the inputs needed for `time_to_reach_kick_position` it runs after all of behavior and hands it's values to the cyclerstate in order for them to be used in the next cycle. As tracked in #402, this cycler state does not support options.
Therefor the time to reach is always Duration::MAX rather than None, and because Lua in the behavior simulator cannot handle duration max it is reduced to 30 minutes.
TLDR We cant have a time to reach of None meaning if everyone is looser everyone claims striker for at least one cycle

**Comment by knoellle** [comment]
`CyclerState` *can* contain an option, but it's impossible to set it to `Some(..)`, this is a framework/code-gen bug
